### PR TITLE
Fetch all usage plan pages

### LINKED ISSUE #257: CatalogUpdaterLambda loads only first 25 usage plans
Lambda catalog-updater uses exports.gateway.getUsagePlans({}) for load usage plans. 
getUsagePlans() function has input parameter:
```
var params = {
  keyId: 'STRING_VALUE',
  limit: 'NUMBER_VALUE',
  position: 'STRING_VALUE'
};
```
By default limit number is 25. (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/APIGateway.html#getUsagePlans-property)

So currently the lambda loads only first 25 usage plans.

### PR DESCRIPTION
**Pull request by alex-chew**
Fixes #257.

Previously, whenever we fetched the list of all usage plans (both in the CatalogUpdaterLambda and in the DevPortalLambda), we only fetched the first page. When there are multiple pages of usage plans, some do not appear in the developer portal frontend, nor in the catalog. This PR resolves that issue.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: lambdas/backend/_common/customers-controller.js [hunk 1]
```diff
@@ -323,16 +319,6 @@ function updateCustomerApiKeyId(cognitoIdentityId, apiKeyId, error, success) {
...
-// function getUsagePlans(error, callback) {
-//     const params = {
-//         limit: 1000
-//     }
-//     apigateway.getUsagePlans(params, (err, data) => {
-//         if (err) error(err)
-//         else callback(data)
-//     })
-// }
-
```
Comment (alex-chew) [inline lambdas/backend/_common/customers-controller.js]: Deleting this snippet because it's clearly not being used, and is obsoleted by this PR.

Comment (Trial-In-Error) [reply lambdas/backend/_common/customers-controller.js]: +1

File: lambdas/shared/get-all-usage-plans.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/**
```
Comment (Trial-In-Error) [inline lambdas/shared/get-all-usage-plans.js]: So, contrary to the title, I'm pretty sure none of these (things in lambdas/shared) are actually shared / used... I don't think there's any action to take on this CR, but [here's an issue for addressing it later](https://github.com/awslabs/aws-api-gateway-developer-portal/issues/287).

Comment (alex-chew) [reply lambdas/shared/get-all-usage-plans.js]: Noted.

**Pull request merged**